code/modules/atmospherics/machinery/components/unary_devices/vent_pump.dm

```diff
@@ -210,10 +210,12 @@
 	if("purge" in signal.data)
 		pressure_checks &= ~EXT_BOUND
 		pump_direction = SIPHONING
+		investigate_log("pump direction was set to [pump_direction] by [key_name(signal_sender)]",INVESTIGATE_ATMOS)
 
 	if("stabilize" in signal.data)
 		pressure_checks |= EXT_BOUND
 		pump_direction = RELEASING
+		investigate_log("pump direction was set to [pump_direction] by [key_name(signal_sender)]",INVESTIGATE_ATMOS)
```

Comment:
```suggestion
		investigate_log("pump direction was set to [pump_direction] by [key_name(signal_sender)]", INVESTIGATE_ATMOS)

	if("stabilize" in signal.data)
		pressure_checks |= EXT_BOUND
		pump_direction = RELEASING
		investigate_log("pump direction was set to [pump_direction] by [key_name(signal_sender)]", INVESTIGATE_ATMOS)
```

if you wanna be real based add a setter for this to hold the duplicate behavior. Don't need to tho

Reply:
I don't really have experience doing that sorry